Header header
uint32 track_id
#view id, use -1 to give label to all views (the only currently implemented)
int32 view_id 
#the category name given by the user
string category_name
